cmd/remove.go

```diff
@@ -34,6 +35,16 @@ func newRemoveCmd() *removeCmd {
 			existingToRemove := []string{}
 
 			for _, p := range args {
+				// If arg is a binary name, find path.
+				if !strings.Contains(p, "/") {
+					for _, b := range cfg.Bins {
+						if b.RemoteName == p {
+							p = b.Path
+							break
+						}
+					}
+				}
+
```

Comment:
Can we move this to a function somewhere and reuse the same logic that's on `update`? Otherwise, if we decide to change the logic eventually, we'd have to do it in two places

Reply:
At first I tried that approach but since [`config` is private](https://github.com/marcosnils/bin/blob/67ed6c35540ae92cd1dd71a5e960c5831126d781/pkg/config/config.go#L17) I decided to copy/paste from `update`.

Do you think is it a valid use-case to make `config` public?

Reply:
> Do you think is it a valid use-case to make `config` public?

No need. We can move the logic from update here https://github.com/marcosnils/bin/blob/master/cmd/update.go#L60-L67 to `root.go` so both `update.go` and `remove.go` can actually use it

Reply:
I'm a little confused, where should I validate args in `root.go`?
I've tried moving it to `Execute()` but it seems it's executed right after the command finishes.

Also, even if I move it to `root.go` it stills needs to get a new `cgf`, in which case I think it would be better to move it to a new file `utils.go` since `root.go` only deals with initial cmd configuration.

Reply:
> Also, even if I move it to `root.go` it stills needs to get a new `cgf`, in which case I think it would be better to move it to a new file `utils.go` since `root.go` only deals with initial cmd configuration.

What I'm basically saying is to create a private package function somewhere ( I don't really see the need to create a file just for this) which performs this name resolution that's both in `update.go` and `remove.go`. We could create a utils.go file as well, even though I don't usually like those type of filenames since they don't represent anything.